OomDetector.cpp

```diff
@@ -46,11 +46,11 @@ bool OomDetector::isOOM(OomdContext& ctx) {
   auto current = Fs::readMemcurrent(cgroup_path_);
```

Comment:
May as well make `current` and `pressure` const as well

Reply:
Done